### PR TITLE
adds more dumb unathi shoes

### PR DESCRIPTION
**Pull request by ParadoxSpace**
adds some more durable caligae as to not literally wear sandals to drop wrenches upon. adds differing tajaran and unathi sprites to fit their different feet. makes unathi feet durable.

### CODE REVIEW

File: code/game/objects/items/weapons/material/shards.dm [hunk 1]
```diff
@@ -68,7 +68,7 @@
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
 
-			if(H.species.siemens_coefficient<0.5 || isvaurca(H) || (H.species.flags & (NO_EMBED))) //Thick skin.
+			if(H.species.siemens_coefficient<0.5 || isunathi(H) || isvaurca(H) || (H.species.flags & (NO_EMBED))) //Thick skin.
```
Comment (skull132) [inline code/game/objects/items/weapons/material/shards.dm]: excuse me.

This is not a documented change.

Comment (VTCobaltblood) [reply code/game/objects/items/weapons/material/shards.dm]: image

Comment (Arrow768) [reply code/game/objects/items/weapons/material/shards.dm]: Still, isnt changelogged

Comment (ParadoxSpace) [reply code/game/objects/items/weapons/material/shards.dm]: handled

**Comment by ParadoxSpace** [comment]
documentation solved

**Comment by ParadoxSpace** [comment]
isn't that good

**Pull request merged**